### PR TITLE
Update lang/pt_br

### PR DESCRIPTION
**Pull request by FITFC**
Updated Brazilian Portuguese to the latest mod texts

### CODE REVIEW

File: common/src/main/resources/assets/voicechat/lang/pt_br.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
 {
   "key.categories.voicechat": "Voice Chat",
-  "key.push_to_talk": "Mantenha pressionado para falar",
+  "key.push_to_talk": "Mantenha pressionado p/ falar",
```
Comment (henkelmax) [inline common/src/main/resources/assets/voicechat/lang/pt_br.json]: Why did you change this?

File: common/src/main/resources/assets/voicechat/lang/pt_br.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
   "key.whisper": "Sussurrar",
   "key.mute_microphone": "Silenciar Microfone",
   "key.disable_voice_chat": "Desabilitar Voice Chat",
   "key.hide_icons": "Ocultar ícones do Voice Chat",
-  "key.voice_chat_settings": "Configurações",
+  "key.voice_chat_settings": "Configs.",
```
Comment (henkelmax) [inline common/src/main/resources/assets/voicechat/lang/pt_br.json]: Why did you change this?

**Review by henkelmax — CHANGES_REQUESTED** [review]

File: common/src/main/resources/assets/voicechat/lang/pt_br.json (hunk 1 above)
Comment (FITFC) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: In Portuguese-Brazilian "Para" can be abbreviated to "p/", this simplifies and does not lose meaning.

File: common/src/main/resources/assets/voicechat/lang/pt_br.json (hunk 2 above)
Comment (FITFC) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: Simplifies and abbreviates the word, without losing the meaning of the word in Portuguese.

Comment (henkelmax) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: Is there any reason to do this? Is the translation too wide to be displayed correctly in game?

File: common/src/main/resources/assets/voicechat/lang/pt_br.json (hunk 1 above)
Comment (henkelmax) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: Is there any reason to do this? Is the translation too wide to be displayed correctly in game?

File: common/src/main/resources/assets/voicechat/lang/pt_br.json (hunk 2 above)
Comment (FITFC) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: This helps to make the text smaller, which makes the buttons prettier and doesn't lose meaning in translation.

File: common/src/main/resources/assets/voicechat/lang/pt_br.json (hunk 1 above)
Comment (FITFC) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: This helps to make the text smaller, which makes the buttons prettier and doesn't lose meaning in translation.

File: common/src/main/resources/assets/voicechat/lang/pt_br.json (hunk 2 above)
Comment (henkelmax) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: This is subjective. As long as there is enough space for translations in the game, there is no need to do this.

Comment (FITFC) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: Do you think it's better to reverse it and leave it without the abbreviation then?

Comment (henkelmax) [reply common/src/main/resources/assets/voicechat/lang/pt_br.json]: Yes

**Comment by henkelmax** [comment]
Thank you!
